docs/resources/cran.md.erb

```diff
@@ -0,0 +1,74 @@
+---
+title: About the cran Resource
+---
+
+# cran
+
+Use the `cran` InSpec audit resource to test R modules that are installed from CRAN package repository.
+
+## Syntax
+
+A `cran` resource block declares a package and (optionally) a package version:
+
+    describe cran('package_name') do
+      it { should be_installed }
+    end
+
+where
+
+* `'package_name'` is the name of the package, such as `'DBI'`
+* `be_installed` tests to see if the package described above is installed
+
+
+## Matchers
+
+This InSpec audit resource has the following matchers:
+
+### be
```

Comment:
Same as with the `cpan.md.erb` file - the boilerplate matcher documentation can be removed from this as well.